doc/manual/manual.tex

```diff
@@ -8037,6 +8038,160 @@ \subsubsection{Melt migration in a 2D mantle convection model}
+A movie of the full model evolution can be found \href{https://www.youtube.com/watch?v=f4Bc4lzdNP0}{online}. 
+
+\paragraph{The input file.}
+One important problem in models with melting and freezing (and other reactions) is that these reactions 
+can be much faster than the time step of the model. For mid-ocean ridges, melt is generally assumed to 
+be in equilibrium with the solid, which means that the reaction is basically instantaneous. 
+To model these type of processes, \aspect{} uses operator splitting (see also Section \ref{sec:benchmark-operator_splitting}): Reactions are solved on a different time scale than advection. 
+For this model, this means that at the beginning of each time step, all melting reactions, 
+including their latent heat effects, are solved using a smaller time step. In the input file, 
+we have to choose both the size of this time step and the rate (or characteristic time scale) of melting, 
+and they have to be consistent in the sense that the operator splitting time step can not be larger than 
+the reaction time scale. 
+The melting model we use here is the anyhdrous mantle melting model of \cite{KSL2003} for a peridotitic 
+rock composition, as implemented in the ``melt simple'' material model. 
+
+\lstinputlisting[language=prmfile]{cookbooks/mid_ocean_ridge/melting_and_freezing.part.prm}
+
+To make sure we reproduce the characteristic triangular melting region of a mid-ocean ridge, we have to  
+set up the boundary conditions in a way that they will lead to corner flow. At the top boundary, we can 
+simply prescribe the half-spreading rate, and at the left boundary we can use a free-slip boundary, as 
+material should not cross this centerline. However, we do not know the inflow and outflow velocities at 
+the bottom and right side of the model. However, what we can do here is prescribing the lithostatic 
+pressure as a boundary condition for the stress. We accomplish this by using the 
+``initial lithostatic pressure'' model. This plugin will automatically compute a 1d lithostatic pressure 
+profile at a given point at the time of the model start and apply it as a boundary traction.
+
+\lstinputlisting[language=prmfile]{cookbooks/mid_ocean_ridge/boundary_conditions.part.prm}
+
+Finally, we have to make sure that the resolution is high enough to model melt migration. 
+This is particularly important in regions where the porosity is low, but still high enough that 
+the two-phase flow equations are solved (instead of the Stokes system, which is solved if there is  
+no melt present in a cell). At the boundary between these regions, material properties like the 
+compaction viscosity may jump, and there may be strong gradients or jumps in some solution variables such 
+as the melt velocity and the compaction pressure. In addition, the characteristic length scale for melt transport, 
+the compaction length $\delta$ depends on the porosity:
```

Comment:
add a comma after "delta$" to make clear that this part is a sub-sentence to "length scale for melt transport"